sdk/src/feature_set.rs

```diff
@@ -123,6 +123,10 @@ pub mod check_program_owner {
     solana_sdk::declare_id!("5XnbR5Es9YXEARRuP6mdvoxiW3hx5atNNeBmwVd8P3QD");
 }
 
+pub mod require_stake_for_gossip {
```

Comment:
Why do we need a feature activation?  It seems to me like this could be rolled out incrementally

Reply:
If half of the cluster upgrade and the other half don't, then the latter will keep sending crds values to the former, but the former ignores those values. Then every pull request will keep sending the same data again, because the first half will just drop those packets. So we will have high network load until the entire cluster upgrades.

Reply:
got it, thanks